### ISSUE TITLE
Data Analysis with Python / Jupyter Notebooks / Twitter Cheat Sheet Broken Link

### ISSUE THREAD

**Issue by trifonas-hash** [freeCodeCamp/freeCodeCamp#41127]
**Describe your problem and how to reproduce it:**
In Data Analysis with Python there are lessons about Jupyter Notebooks. And on the bottom side of the videos there are some useful links. The last one is a link to Twitter posts that contain various tips, commands etc for Jupyter.
Well, that link doesn’t work anymore (I think the lesson was made in like March 2020).

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/how-to-use-jupyter-notebooks-intro
https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells
https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-importing-and-exporting-data

and the supposed link: https://twitter.com/rmotr_com/status/1122176794696847361

**Recommended fix, suggestions (how would you update it?):**
If the cheat sheet still exists, then we can update the link. Else, it should be deleted from the lessons.

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
![Screenshot_3](https://user-images.githubusercontent.com/76069062/107946047-2bc5e100-6f99-11eb-8add-7ba841a7bd65.png)

**Comment by JoshuaPelealu** [comment]
It seems like the tweet is gone. When I check out the user, it looks like they do not have any tweets... They might have deleted their contents

**Comment by raisedadead** [comment]
We should remove these links, @scissorsneedfoodtoo or @beaucarnes would either of you be able to create a news article that could be used here instead?

**Comment by JoshuaPelealu** [comment]
I can take care of removing the links, but should I wait until a new link is done, or should I get a pull request?

**Comment by raisedadead** [comment]
> ... but should I wait until a new link is done, or should I get a pull request?

I would say, let's wait for either Beau or Kris to respond back. Thanks for your interest in making a PR, have been looking at some of your recent contributions. Thanks for your time and keep up the good work!

**Comment by raisedadead** [comment]
Hey @JoshuaPelealu, Beau says you can proceed with removing that link all together for now. We will add a new link in a later PR.

Thanks again for working on this.

**Comment by JoshuaPelealu** [comment]
Should the PR close this issue, or just let the issue stand until the later PR for the new links?

**Comment by scissorsneedfoodtoo** [comment]
Thanks again for your quick fix, @JoshuaPelealu, and for reporting this @trifonas-hash.

I talked this over with @beaucarnes and we decided that it's not necessary to replace the broken link. The current links should be enough for learners to get started with Colaboratory. Also, the broken link was probably a video walkthrough of the content in the second link in those challenges, [How to open Notebooks from GitHub using Google Colab](https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb).
